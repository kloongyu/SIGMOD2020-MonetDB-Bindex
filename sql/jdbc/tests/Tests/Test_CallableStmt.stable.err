stderr of test 'Test_CallableStmt` in directory 'sql/jdbc/tests` itself:


# 19:25:40 >  
# 19:25:40 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=36596" "--set" "mapi_usock=/var/tmp/mtest-14657/.s.monetdb.36596" "--set" "monet_prompt=" "--forcemito" "--dbpath=/export/scratch1/dinther/INSTALL/var/MonetDB/mTests_sql_jdbc_tests" "--set" "embedded_c=true"
# 19:25:40 >  

# builtin opt 	gdk_dbpath = /export/scratch1/dinther/INSTALL/var/monetdb5/dbfarm/demo
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 36596
# cmdline opt 	mapi_usock = /var/tmp/mtest-14657/.s.monetdb.36596
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbpath = /export/scratch1/dinther/INSTALL/var/MonetDB/mTests_sql_jdbc_tests
# cmdline opt 	embedded_c = true
#main thread:!ERROR:MALException:client.quit:Server stopped

# 19:25:40 >  
# 19:25:40 >  "./Test_CallableStmt.SQL.sh" "Test_CallableStmt"
# 19:25:40 >  


# 19:25:40 >  
# 19:25:40 >  java Test_CallableStmt "jdbc:monetdb://catskill:36596/mTests_sql_jdbc_tests?user=monetdb&password=monetdb"
# 19:25:40 >  


# 19:25:40 >  
# 19:25:40 >  "Done."
# 19:25:40 >  

