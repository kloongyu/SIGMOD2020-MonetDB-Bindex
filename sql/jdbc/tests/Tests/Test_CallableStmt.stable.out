stdout of test 'Test_CallableStmt` in directory 'sql/jdbc/tests` itself:


# 19:25:40 >  
# 19:25:40 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=36596" "--set" "mapi_usock=/var/tmp/mtest-14657/.s.monetdb.36596" "--set" "monet_prompt=" "--forcemito" "--dbpath=/export/scratch1/dinther/INSTALL/var/MonetDB/mTests_sql_jdbc_tests" "--set" "embedded_c=true"
# 19:25:40 >  

# MonetDB 5 server v11.33.0 (hg id: 70b8126598ab+)
# This is an unreleased version
# Serving database 'mTests_sql_jdbc_tests', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 31.305 GiB available main-memory.
# Copyright (c) 1993 - July 2008 CWI.
# Copyright (c) August 2008 - 2019 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://catskill.da.cwi.nl:36596/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-14657/.s.monetdb.36596
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

Ready.

# 19:25:40 >  
# 19:25:40 >  "./Test_CallableStmt.SQL.sh" "Test_CallableStmt"
# 19:25:40 >  


# 19:25:40 >  
# 19:25:40 >  java Test_CallableStmt "jdbc:monetdb://catskill:36596/mTests_sql_jdbc_tests?user=monetdb&password=monetdb"
# 19:25:40 >  

Created table: tbl6402
Created procedure: proc6402
Called procedure (1): proc6402
Table tbl6402 has 7 columns:
	tint	tdouble	tbool	tvarchar	tclob	turl	tclen
	1	1.1	true	one	ONE	www.monetdb.org	6
Prepared Callable procedure: proc6402
Called Prepared procedure (1): proc6402
6 parameters reported:
1.	nullable  2 (UNKNOWN)
	signed    true
	precision 32
	scale     0
	type      4
	typename  int
	classname java.lang.Integer
	mode      1 (IN)
2.	nullable  2 (UNKNOWN)
	signed    true
	precision 53
	scale     0
	type      8
	typename  double
	classname java.lang.Double
	mode      1 (IN)
3.	nullable  2 (UNKNOWN)
	signed    false
	precision 1
	scale     0
	type      16
	typename  boolean
	classname java.lang.Boolean
	mode      1 (IN)
4.	nullable  2 (UNKNOWN)
	signed    false
	precision 15
	scale     0
	type      12
	typename  varchar
	classname java.lang.String
	mode      1 (IN)
5.	nullable  2 (UNKNOWN)
	signed    false
	precision 0
	scale     0
	type      2005
	typename  clob
	classname java.sql.Clob
	mode      1 (IN)
6.	nullable  2 (UNKNOWN)
	signed    false
	precision 0
	scale     0
	type      12
	typename  url
	classname nl.cwi.monetdb.jdbc.types.URL
	mode      1 (IN)
Table tbl6402 has 7 columns:
	tint	tdouble	tbool	tvarchar	tclob	turl	tclen
	1	1.1	true	one	ONE	www.monetdb.org	6
	2	2.02	true	Two	TWOs	http://www.monetdb.org/	7
Called Prepared procedure (2): proc6402
Table tbl6402 has 7 columns:
	tint	tdouble	tbool	tvarchar	tclob	turl	tclen
	1	1.1	true	one	ONE	www.monetdb.org	6
	2	2.02	true	Two	TWOs	http://www.monetdb.org/	7
	2	3.02	true	Tree	TWOs	https://www.monetdb.org/	8
Called Prepared procedure (3): proc6402
Table tbl6402 has 7 columns:
	tint	tdouble	tbool	tvarchar	tclob	turl	tclen
	1	1.1	true	one	ONE	www.monetdb.org	6
	2	2.02	true	Two	TWOs	http://www.monetdb.org/	7
	2	3.02	true	Tree	TWOs	https://www.monetdb.org/	8
	4	3.02	false	Four	TWOs	https://www.monetdb.org/	8
Called Prepared procedure (with NULLs): proc6402
Table tbl6402 has 7 columns:
	tint	tdouble	tbool	tvarchar	tclob	turl	tclen
	1	1.1	true	one	ONE	www.monetdb.org	6
	2	2.02	true	Two	TWOs	http://www.monetdb.org/	7
	2	3.02	true	Tree	TWOs	https://www.monetdb.org/	8
	4	3.02	false	Four	TWOs	https://www.monetdb.org/	8
	null	null	null	null	null	https://www.monetdb.org/	null
Test completed. Cleanup procedure and table.

# 19:25:40 >  
# 19:25:40 >  "Done."
# 19:25:40 >  

